packages/virtual-list/test/lit-renderer.test.js

```diff
@@ -0,0 +1,83 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
+import '../vaadin-virtual-list.js';
+import { html, render } from 'lit';
+import { virtualListRenderer } from '../lit.js';
+
+async function renderVirtualList(container, { items }) {
+  render(
+    html`
+      <vaadin-virtual-list
+        .items="${items}"
+        ${items ? virtualListRenderer((item) => html`${item}`, items) : null}
+      ></vaadin-virtual-list>
+    `,
+    container,
+  );
+  await nextFrame();
+  return container.querySelector('vaadin-virtual-list');
+}
+
+describe('lit renderer directive', () => {
```

Comment:
Let's call the file `lit-renderer-directives.test.js` to keep it in line with other packages.

Reply:
Done.